README.adoc

```diff
@@ -163,76 +162,55 @@ http://json-b.net/users-guide.html[JSON Binding 1.0 Users Guide].
 
 == Consuming the REST resource
 
-Write a simple `Consumer` class that consumes the artist JSON using both JSON-B and JSON-P.
-
-`src/main/java/io/openliberty/guides/consumingrest/Consumer.java`:
+The `Artist` and `Album` POJOs are ready for deserialization. To consume the JSON response
+from your REST service create the `Consumer` class in the
+`src/main/java/io/openliberty/guides/consumingrest/Consumer.java` file.
 
 [source, java, indent=0]
 ----
-include::finish/src/main/java/io/openliberty/guides/consumingrest/Consumer.java[tags=!comment;!class]
+include::finish/src/main/java/io/openliberty/guides/consumingrest/Consumer.java[tags=!comment;]
```

Comment:
@evelinec Combined the code snippets into this one area as instructed by issue here: https://github.com/OpenLiberty/guide-rest-client-java/issues/23